pkg/cmd/aries-agentd/main.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// Package aries-agentd (Aries Agent Server) of aries-framework-go.
+//
+//
+// Terms Of Service:
+//
+// there are no TOS at this moment
+//
+//     Schemes: http, https
+//     Host: 127.0.0.1:8080
+//     Version: 0.1.0
+//     License: Copyright SecureKey Technologies Inc. All Rights Reserved.
+//
+//     Consumes:
+//     - application/json
+//
+//     Produces:
+//     - application/json
+//
+// swagger:meta
+package main
+
+import (
+	"net/http"
+	"os"
+
+	"github.com/go-openapi/runtime/middleware/denco"
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
+	"github.com/hyperledger/aries-framework-go/pkg/framework/aries"
+)
+
+const agentHostEnvKey = "ARIESD_API_HOST"
+
+var logger = log.New("aries-framework/agentd")
+
+// This is an application which starts Aries agent controller API on given port
+func main() {
+
+	host := os.Getenv(agentHostEnvKey)
+	if host == "" {
+		logger.Errorf("Unable to start aries agentd, host not provided")
```

Comment:
Any reason why we can't have a default?

Reply:
Since there is no stop, we need to know the port to kill it.
I am still thinking about default or start on any port available if not provided, which will be done as part of #94

Reply:
(for #94, look at cobra).